src/test/java/com/zufar/onlinestore/product/endpoint/e2e/ProductsEndpointTest.java

```diff
@@ -0,0 +1,80 @@
+package com.zufar.onlinestore.product.endpoint.e2e;
+
+import com.zufar.onlinestore.test.config.AbstractE2ETest;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import static com.zufar.onlinestore.test.config.RestAssertion.*;
+import static io.restassured.RestAssured.given;
+
+
+@Testcontainers
+class ProductsEndpointTest extends AbstractE2ETest {
```

Comment:
Add `@DisplayName("ProductsEndpoint Tests")
` annotation on the class level, please